pom.xml

```diff
@@ -450,7 +452,7 @@
         <profile>
             <id>build-with-spotbugs</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <activeByDefault>false</activeByDefault>
```

Comment:
I've turned this off by default, as it 1/ helps with the SAM tooling for the examples and 2/ speeds up local builds that haven't opted into it.

The github workflows explicitly activate and de-activate these using the profiles, so it has no impact on CI.